events/story_cycles/wc_story_cycle_horde_invasion_events.txt

```diff
@@ -89,28 +200,104 @@ WCHOI.3 = {
 	}
 	left_portrait = scope:warchief
 
-	# The orcs need their warchief!
-	option = {
-		trigger = {
-			OR = {
-				culture_group = culture_group:orc_group
-				root = scope:warchief
+	option = {	# Stay
```

Comment:
This option doesn't include stress gain or loss.

Reply:
Added